src/scripts/content.ts

```diff
@@ -33,7 +33,11 @@ const connection = browser.runtime.connect(browser.runtime.id, {
 // forward messages from the api to the background script
 window.addEventListener("message", async (e) => {
   if (!validateMessage(e.data, {}) || !e.data.type) return;
+  let id = e.data.id;
```

Comment:
We don't need a variable for this, let's remove it

Reply:
done!